code/controllers/subsystem/supply_shuttle.dm

```diff
@@ -169,6 +169,12 @@ var/datum/subsystem/supply_shuttle/SSsupply_shuttle
 		if(O.CheckShuttleObject(A,in_crate,preserve))
 			return
 
+/datum/subsystem/supply_shuttle/proc/scrub()
```

Comment:
I don't think there will be any unforeseen consequences if you just do this for all obj/effect's - this will also remove things like metal foam or any fires on the shuttle

Reply:
I'd rather not, that includes non-cleanable decals and a lot of stuff that shouldn't disappear.